pkgs/servers/nosql/influxdb2/default.nix

```diff
@@ -43,6 +43,11 @@ let
     sourceRoot = "source/libflux";
     cargoSha256 = "sha256-+hJQFV0tWeTQDN560DzROUNpdkcZ5h2sc13akHCgqPc=";
     nativeBuildInputs = [ llvmPackages.libclang ];
+    postPatch = ''
```

Comment:
this is not needed if you rebase on latest master

Reply:
Thanks, removed postPatch